### PR TITLE
Adds auto create webhook for cockpit

### PR DESCRIPTION
**Pull request by ashraffouda**
Signed-off-by: Ashraf Fouda <foudaa@codescalers.com>

### CODE REVIEW

File: ays/github_repo/actions.py [hunk 1]
```diff
@@ -143,9 +143,21 @@ def init(self, service):
         clienthrd.set("code.path", j.dirs.replaceTxtDirVars(clienthrd.get("code.path")))
 
         path = j.sal.fs.joinPaths(clienthrd.get("code.path"), service.hrd.get("repo.account"), service.hrd.get("repo.name"))
-
         service.hrd.set("code.path", path)
+        cockpit_repo = j.atyourservice.get("ays_cockpit", "/opt/code/cockpit/ays_cockpit")
+        bp_service = cockpit_repo.getService("blueprint", "pilot")
```
Comment (zaibon) [inline ays/github_repo/actions.py]: This is not correct service to read from.
service name is `os.cockpit` and the instance name is not know in advance. so look for the service name like that `cockpit_repo.findServices(templatename='os.cockpit')` there is only one service of this type in the repo.

File: ays/github_repo/actions.py [hunk 2]
```diff
@@ -143,9 +143,21 @@ def init(self, service):
...
+        dns_domain = os_cockpit_service.hrd.get("dns.domain")
+        webhook_url = "{dns_domain}/api/webhooks/github".format(dns_domain=dns_domain)
+        repo = self.get_github_repo(service)
 
+        for hook in repo.api.get_hooks():
+            if hook.config['url'] == webhook_url:
+                break
+        else:
+            config = {"url": webhook_url, "content_type": "json"}
+            repo.api.create_hook("web", config)
```
Comment (zaibon) [inline ays/github_repo/actions.py]: maybe give a more meaning full name for the webhook. web is very vague.

Comment (ashraffouda) [inline ays/github_repo/actions.py]: web is used in the api as key every web hook should use "web" as a name

Comment (zaibon) [inline ays/github_repo/actions.py]: Ignore my comment then

**Pull request merged**